### PR TITLE
build(Package scripts): Remove unused scripts

### PR DESCRIPTION
**Pull request by tcrosado**
Removed unnecessary npm script no longer used.

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [x] Code is well-commented, linted and follows project conventions
- [x] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

**Review by GustavoRSSilva — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -35,10 +35,7 @@
     "address": "truffle exec scripts/address-proxy.js",
```
Comment (DioPires) [inline package.json:35]: #### Tests:
```diff 
- 4 failing
```
```diff 
+ 100 passing
```
#### Test coverage: 
81.44 %

Comment (DioPires) [inline package.json:35]: #### Tests:
```diff 
- 4 failing
```
```diff 
+ 100 passing
```
#### Test coverage: 
81.44 %

Comment (DioPires) [inline package.json:35]: #### Tests:
```diff 
- 4 failing
```
```diff 
+ 100 passing
```
#### Test coverage: 
81.44 %

**Pull request merged**